net-misc/zerotier/zerotier-1.2.8.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic systemd toolchain-funcs
+
+HOMEPAGE="https://www.zerotier.com/"
+DESCRIPTION="A software-based managed Ethernet switch for planet Earth"
+SRC_URI="https://github.com/zerotier/ZeroTierOne/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/ZeroTierOne-${PV}"
+
+RDEPEND="
+	dev-libs/json-glib
+	net-libs/http-parser
+	net-libs/libnatpmp
+	net-libs/miniupnpc"
+
+DEPEND="${RDEPEND}"
+
+QA_PRESTRIPPED="/usr/sbin/zerotier-one"
```

Comment:
Fix the build system not to strip instead. I think passing `STRIP=:` to emake will work here.